src/bindings/jna/README.md

```diff
@@ -17,27 +20,48 @@ then you can compile and run [HelloElektra](HelloElektra.java):
 
     javac HelloElektra.java && java HelloElektra
 
-For plugin development, see [plugins](elektra/plugin)
+Alternatively you can also specify the classpath directly, both during compilation and execution. Note that in this example, we have the elektra and jna libraries in different directories, as mac doesn't allow writing to /usr/share/java in never versions. Also note its important in that case to include this directory containing the compiled HelloElektra.class when executing it, otherwise it will not be found:
+
+	javac -cp ./libelektra4j.jar:./libelektra4j/jna.jar HelloElektra.java
+	java -cp .:./libelektra4j.jar:./libelektra4j/jna.jar HelloElektra
+
+For plugin development, see [plugins](libelektra4j/plugin)
 and also [here](/src/plugins/jni) for more information.
 
+### Maven ###
+
+In case you want to use maven as your build system, simply go into the libelektra4j subfolder and execute mvn install. This will install the plugin into your local maven repository. Afterwards you can include it as a dependency into your maven project, and given that libelektra is actually installed on your system you can use it by including the maven dependency:
+	<groupId>org.libelektra</groupId>
+	<artifactId>libelektra4j</artifactId>
+	<version>0.0.1-SNAPSHOT</version>
```

Comment:
0.0.1-SNAPSHOT the default for new maven projects. I agree that a higher version number makes sense, but I'm not sure if it really makes that much sense to use the same as elektra because this introduces the problem of having to synchronize this version number with elektra's one somehow to stay consistent in future releases. But if there are no changes to the jna binding for a new elektra version, does it make sense to update the version number nevertheless? Since it looks like some things are not really perfect yet with the jna bindings (see the readme) i'd use something like 0.9.0 or similar for now.

SNAPSHOT is some maven convention, usually you'd use -SNAPSHOT for development versions and then switch to -RELEASE for, well, release versions. It is not required to call it like that though, so I could drop it completely if you prefer.